drivers/bmp180/bmp180_saul.c

```diff
@@ -39,9 +39,9 @@ static int read_pressure(void *dev, phydat_t *res)
 {
     bmp180_t *d = (bmp180_t *)dev;
 
-    res->val[0] = bmp180_read_pressure(d) / 10;
+    res->val[0] = bmp180_read_pressure(d) / 100;
```

Comment:
There is a one digit precision loss if you do that. The previous division already introduced one.